### PR TITLE
Adding Secondary on Review Error

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
- Fix error on review when adding/removing secondary caregiver from review panel

## Screenshots
image

## Acceptance criteria
- [x] When adding Secondary Caregiver on review page for the first time does not cause error

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import ErrorableTextInput from '@department-of-veterans-affairs/formation-react/
 
 const SignatureInput = ({ fullName, required, label, setIsSigned }) => {
   const [hasError, setError] = useState(false);
-  const firstName = fullName.first.toLowerCase();
-  const lastName = fullName.last.toLowerCase();
-  const middleName = fullName.middle === undefined ? '' : fullName.middle;
+  const firstName = fullName.first ? fullName.first.toLowerCase() : '';
+  const lastName = fullName.last ? fullName.last.toLowerCase() : '';
+  const middleName = fullName.middle ? fullName.middle : '';
```
Comment (Mottie) [inline src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx:8]: Optional chaining could be used here!

```js
const firstName = fullName.first?.toLowerCase() || '';
```

**Review by Mottie — APPROVED** [review]

File: src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx (hunk 1 above)
Comment (LinnJS) [reply src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx:8]: I was thinking about optional chaining but it just felt like more code that was not necessarily needed. Is there any benefit to this @Mottie

Comment (Mottie) [reply src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx:8]: Benefit? I can't think of any... is it purdy? yes 😉

Comment (LinnJS) [reply src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx:8]: After thinking about it seems more readable so gonna go with that

**Pull request merged**